### PR TITLE
side quest: Like & Re-Tweet oss.gg Launch Tweet

### PR DESCRIPTION
**Pull request by Naprila**
Description:
Liked & Tweeted @twentycrm on X

Points:
50

Proof:
Link: https://x.com/mkprasad_821/status/1847886807314120762

Screenshot 2024-10-20 at 11 56 54 AM

### CODE REVIEW

File: oss-gg/twenty-side-quest/1-quote-tweet-20-oss-gg-launch.md [hunk 1]
```diff
@@ -46,3 +46,6 @@ Your turn 👇
 
 » 16-October-2024 by Harsh Bhat
 » Link to Tweet: https://x.com/HarshBhatX/status/1846252536241508392
+
+» 20-October-2024 by Naprila
+» Link to Tweet: https://x.com/mkprasad_821/status/18478868073141207622
```
[the remark this answers is not in the document]
Comment (Naprila) [reply oss-gg/twenty-side-quest/1-quote-tweet-20-oss-gg-launch.md]: Fixed the tweed ID

**Review by charlesBochet — APPROVED** [review]
Ty!

**Pull request merged**